### PR TITLE
[behatsdaa] Fix amount to be negative

### PR DESCRIPTION
**Pull request by daniel-hauser**
The current implementation of the scraper takes the amount from the `customerPrice` field returned from the behatsdaa api result,  which has the charged amount as a positive number.

This PR makes it negative as it's an expense

### CODE REVIEW

**Comment by daniel-hauser** [comment]
@esakal I believe the same issue might be in the **beyahad-bishvilha** scraper, but i have no account to verify.

File: src/scrapers/behatsdaa.ts [hunk 1]
```diff
@@ -32,14 +32,15 @@ type PurchaseHistoryResponse = {
 };
 
 function variantToTransaction(variant: Variant): Transaction {
+  const originalAmount = -variant.customerPrice;
```
Comment (daniel-hauser) [inline src/scrapers/behatsdaa.ts:35]:
```suggestion
  const originalAmount = -variant.customerPrice;
  
```

File: src/scrapers/behatsdaa.ts [hunk 2]
```diff
@@ -32,14 +32,16 @@ type PurchaseHistoryResponse = {
...
+
```
Comment (daniel-hauser) [inline src/scrapers/behatsdaa.ts:36]:
```suggestion
```

File: src/scrapers/behatsdaa.ts (hunk 1 above)
Comment (daniel-hauser) [inline src/scrapers/behatsdaa.ts:35]:
```suggestion
  // The price is positive, make it negative as it's an expense
  const originalAmount = -variant.customerPrice;
```

**Review by baruchiro — APPROVED** [review]

**Comment by baruchiro** [comment]
Hi @daniel-hauser, [Call for Code Owners](https://github.com/eshaham/israeli-bank-scrapers/issues/830)

**Comment by daniel-hauser** [comment]
@baruchiro - With the power granted to me in #830, i hereby declare this PR as tested 😉, can you merge it?

**Comment by baruchiro** [comment]
Ha ha ha @daniel-hauser , the purpose is to get at least two testers for a change, to make sure your change is not over fitting for you. 😀

**Pull request merged**